README.md

```diff
@@ -16,14 +16,26 @@ To be able to use XenIssuing, you will need to use a private key provided by Xen
 XenCrypt is a module to help you set up encryption between XenIssuing and your application.
 
 It includes several methods:
-- `generateSessionId` will encrypt a session key randomly generated used for symmetric encryption with Xenissuing.
+- `generateSessionId` will encrypt a session key randomly generated used for asymmetric encryption with Xenissuing.
 - `encrypt` would be used when setting sensitive data.
 - `decrypt` would be used whenever receiving sensitive data from Xenissuing.
 
+
+### Session ID
+
+```swift
+let xen = try! Xenissuing(xenditPublicKeyData: Data(base64Encoded: "BASE64_PUBLIC_KEY")!)
+let sessionKey = try! xcrypt.generateRandom()
```

Comment:
is this suppose to be `xcrypt` or `xen`?

Reply:
correct, typo thanks 🙏🏽